client/web/src/site-admin/SiteAdminRepositoriesPage.tsx

```diff
@@ -87,6 +88,13 @@ const RepositoryNode: React.FunctionComponent<React.PropsWithChildren<Repository
                 </Alert>
             </div>
         )}
+        {node.mirrorInfo.corruptedAt && (
+            <div className={styles.alertWrapper}>
+                <Alert variant="warning">
```

Comment:
```suggestion
                <Alert variant="error">
```

I think this warrants an error, right?